courses/tests.py

```
@@ -1,3 +1,86 @@
+class EnrollmentTest(TestCase):
+    def setUp(self):
+        self.course = Course.objects.create(title=COURSE_TITLE)
+        self.student = User.objects.create(username="student_user")
+        self.enrollment = Enrollment.objects.create(course=self.course, student=self.student, status=ENROLLMENT_STATUS)
+    
+    def test_enrollment_creation(self):
+        self.assertEqual(self.enrollment.course, self.course)
+        self.assertEqual(self.enrollment.student, self.student)
+        self.assertEqual(self.enrollment.status, ENROLLMENT_STATUS)
+    
+    def test_enrollment_increments_course_enrollment_count(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_enrollment_increments_course_enrollment_count`

Add return type annotation: `None`

(ANN201)